Sources/Clappr/Classes/Plugin/Core/MediaControl/MediaControl.swift

```diff
@@ -0,0 +1,236 @@
+import Foundation
+
+class MediaControl: UICorePlugin, UIGestureRecognizerDelegate {
```

Comment:
If we're going to support 3d party `MediaControl` again I highly suggest to make `MediaControl` a more generic class.

Then extend it using this one.